g/grpc-go/grpc-go_ubi_8.4.sh

```diff
@@ -0,0 +1,59 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : grpc-go 
+# Version       : v1.16.0, master
+# Source repo   : https://github.com/grpc/grpc-go 
+# Tested on     : UBI 8.4
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Nagesh Tarale <Nagesh.Tarale@ibm.com>
+#
+# Disclaimer: This script has been tested in non-root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#!/bin/bash
+
+if [ -z "$1" ]; then
+  export VERSION=master
+else
+  export VERSION=$1
+fi
+
+if [ -d "grpc-go" ] ; then
+  rm -rf grpc-go
+fi
+
+# Dependency installation
+dnf update
+dnf install -y go
+dnf install -y git
+dnf install -y wget
+
+# Download the repos
+git clone https://github.com/grpc/grpc-go
+
+cd grpc-go 
+git checkout $VERSION
+
+ret=$?
+
+if [ $ret -eq 0 ] ; then
+ echo "$Version found to checkout "
+else
+ echo "$Version not found "
+ exit
+fi
+
+#Installation
+go get -u grpc-go 
+
+#Build
+go mod tidy
+go mod vendor
+go build -mod=vendor
```

Comment:
Can we capture the build status and exit if it fails instead of continuing with tests ?

Reply:
ok